cli/main.go

```diff
@@ -130,23 +134,38 @@ func Main() {
 	log.Debug("debug logging enabled")
 
 	genesisForkVersionHex := ""
+	var genesisTime uint64
+
 	switch {
 	case *useCustomGenesisForkVersion != "":
 		genesisForkVersionHex = *useCustomGenesisForkVersion
-	case *useGenesisForkVersionSepolia:
+	case *sepolia:
 		genesisForkVersionHex = genesisForkVersionSepolia
-	case *useGenesisForkVersionGoerli:
+	case *goerli:
 		genesisForkVersionHex = genesisForkVersionGoerli
-	case *useGenesisForkVersionZhejiang:
-		genesisForkVersionHex = genesisForkVersionZhejiang
-	case *useGenesisForkVersionMainnet:
+	case *mainnet:
 		genesisForkVersionHex = genesisForkVersionMainnet
 	default:
 		flag.Usage()
-		log.Fatal("please specify a genesis fork version (eg. -mainnet / -sepolia / -goerli / -zhejiang / -genesis-fork-version flags)")
+		log.Fatal("please specify a genesis fork version (eg. -mainnet / -sepolia / -goerli / -genesis-fork-version flags)")
 	}
 	log.Infof("using genesis fork version: %s", genesisForkVersionHex)
 
+	switch {
```

Comment:
don't add a second switch here, assign the time in the switch above, and then below just do a simple `if *useCustomGenesisTime > -1`